main.tex

```diff
@@ -704,11 +699,11 @@ \subsection*{Re-visitable fields}
 \includegraphics[scale=0.8]{images/black_market.PNG}\\
 \includegraphics[scale=0.8]{images/sanctuary.PNG}
 \includegraphics[scale=0.8]{images/tavern.PNG}\\
```

Comment:
```suggestion
\includegraphics[scale=0.8]{images/black_market.PNG}
\includegraphics[scale=0.8]{images/sanctuary.PNG}
\includegraphics[scale=0.8]{images/tavern.PNG}
```
Are those leftovers?

Reply:
I wouldn't touch these ones yet. A \\ is a regular, shorter line break and they are here currently just to rearrange the pictures in some way to fit them onto the page. Removing them would slide some of the pictures so that only half of them are visible.